pkgs/by-name/pi/pinball/package.nix

```diff
@@ -30,26 +33,23 @@ stdenv.mkDerivation rec {
   ];
   buildInputs = [
     libglvnd
-    SDL
-    SDL_image
-    SDL_mixer
+    libtool
```

Comment:
Isn't libtool part of autoreconfHook?

Reply:
It needs it at runtime (for libltdl), i.e. as a build input, not a native build input.